core/src/test/scala/com/pingcap/tispark/datasource/DataTypeSuite.scala

```diff
@@ -0,0 +1,432 @@
+package com.pingcap.tispark.datasource
+
+import java.sql.{Date, Timestamp}
+import java.util.Calendar
+
+import com.pingcap.tikv.exception.TiBatchWriteException
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.types.{ByteType, CharType, DecimalType, DoubleType, FloatType, IntegerType, LongType, StringType, StructField, StructType, TimestampType, VarcharType}
+
+
+class DataTypeSuite extends BaseDataSourceTest("test_datasource_datatype") {
```

Comment:
maybe should delete this file `com.pingcap.tispark.datatype.DataTypeSuite`

Reply:
done